pkg/backend/v1/sqlite/schema_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/canonical/k8s-dqlite/pkg/drivers/sqlite"
+	sqlite "github.com/canonical/k8s-dqlite/pkg/backend/v1/sqlite"
```

Comment:
is a named import necessary? Souldn't it already be called "sqlite" by default? This applies everywhere